src/index.html

```diff
@@ -11,7 +11,149 @@
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label class="form__field">
+          How old are You?
+          <input 
+            type="number" 
+            name="howOldAreYou"
+            value="12"
+            min="1"
+            max="100"
+            required
+          >
+        </label>
+        <label class="form__field">
+          Full date of birth:
+          <input
+            type="date" 
+            name="date"
+            required
+          >
+        </label>
+        <label class="form__field-last">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="acceptAgreement" 
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form__field">
+          E-mail:
+          <input
+            type="email" 
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form__field-last">
+          Password:
+          <input 
+            type="password" 
+            name="pass"
+            minlength="5"
+            maxlength="30"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form__field">
+          Do you love cats?
+          <input 
+            type="radio" 
+            name="cats"
+          >
+          Yes
+          <input 
+            type="radio" 
+            name="cats"
+          >
+          No
+        </label>
+        <label class="form__field">
+          What is your favorite color?
+          <input 
+            type="color"
+            name="favorite"
+          >
+        </label>
+        <label class="form__field">
+          What time do you go to bed?
+          <input 
+            type="time" 
+            name="timeToBed"
+          >
+        </label>
+        <label class="form__field">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="form__field-last">
+          How do you rate out work?
+          <input 
+            type="range" 
+            name="rateOutWork"
+            min="0"
+            max="15"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <label class="form__field">
+          Comments:
+          <textarea 
+            name="comments"  
+            cols="20" 
+            rows="2"
+            required
+            minlength="5"
+            maxlength="100"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
to prevent this
<img width="316" alt="image" src="https://user-images.githubusercontent.com/53611910/206903148-4fc68abf-1127-4f6b-9b39-eb25a28861b5.png">